packages/server-common/test/handlers/calendar/subscribe-handler.test.ts

```diff
@@ -477,6 +477,55 @@ describe('Subscription', () => {
     });
     expect(nock.isDone()).toBe(true);
   });
+
+  test('Should subscribe to the calendar events notifications and return the resourceId when passing an additional path', async () => {
```

Comment:
I still don't fully understand what is going on - isn't this handler ONLY for squidex?

Reply:
I think what was confusing is that this `packages/server-common/src/handlers/calendar/subscribe-handler.ts` file contains 3 functions
- `subscribeToEventChangesFactory`
- `unsubscribeFromEventChangesFactory`
- `calendarCreatedHandlerFactory`

but
- `subscribeToEventChangesFactory`
- `unsubscribeFromEventChangesFactory`

were used by both Contentful and Squidex calendar handler factory. So I decided to split the files, now `packages/server-common/src/handlers/calendar/subscribe-handler.ts` only contains
- `subscribeToEventChangesFactory`
- `unsubscribeFromEventChangesFactory`

and `calendar-handler-factory.squidex` and `calendar-handler-factory.contentful` contains their calendar handler factory